legacy/monitor.ts

```diff
@@ -1,9 +1,10 @@
-import { DepTree } from './common';
+import { DepTree, DepGraph } from './common';
 
 export interface MonitorBody {
   meta: MonitorMeta;
   policy: string;
-  package: DepTree;
+  package?: DepTree;
+  packageGraph?: DepGraph;
```

Comment:
maybe use this chance to stop calling it a package?

Reply:
we can do that in a side pr, we need to consider all the other projects that uses this interface before of do it

Reply:
was refering to the `packageGraph` - nothing uses it yet

Reply:
I understand, I wanted it concise with the existing `package`. I think we can move on with this naming and then refactor both with a valid name that makes sense for all ecosystem

Reply:
imo `dependecyGraph` is better

and I think it should be in this PR :)

Reply:
Changed to dependencyGraph, if this name is not enough we can change it later.